MIGRATION.md

```diff
@@ -1,37 +1,26 @@
-# 1.1.0
+# v2.0.0
 
-As of `logr@v1.0.0`, the `logr.Logger` is considered to be a defined `struct` instead of an `interface`. The implementation layer (now referred to as `logr.LogSink`) has been entirely restructured. Now, the `logerr` library will provide `logr.Logger` objects and ways to affect the underlying `Sink` operations.
+## Logger creation
 
-- Instead of `log.V()`, `log.Info()`, `log.Error()` methods, use `log.DefaultLogger().V()`, `log.DefaultLogger().Info()`, `log.DefaultLogger().Error()`
+`NewLoggerWithOptions` has been removed in favor of a singular `NewLogger` method which accepts a string and array of options as an argument. Setting keyValuePairs as part of the initialization has been removed. Instead, use the `WithValues` method to add more keyValuePairs to the logger.
 
-- Instead of `log.SetOutput()`, `log.SetLogLevel()`, use either of the following methods:
+## Removal of singleton behavior
 
-Method 1:
-```go
-l := log.DefaultLogger()
-s, err := log.GetSink(l)
+Due to the structural changes of `logr@v1`, several features of this library have been modified. Most notable of these changes is the lack of a singleton logger. It is recommended that developers create a logger instance with `NewLogger` and either keep it as their own singleton or pass the instance throughout the application to use where needed. Methods like `Info` and `Error` are still callable with a `logr.Logger` instance.
 
-if err != nil {
-    // Some action
-}
+ex:
 
-s.SetVerbosity(1)
-s.SetOutput(ioutil.Discard)
+```golang
+import (
+    "github.com/ViaQ/logerr/v2/log"
+)
 
-l.Info("hello world")
-```
-
-Method 2:
-```go
-l := log.DefaultLogger()
-// This method panics, but DefaultLogger will not
-// panic because it uses Sink.
-log.MustGetSink(l).SetVerbosity(1)
-log.MustGetSink(l).SetOutput(ioutil.Discard)
+logger := log.NewLogger("", nil)
```

Comment:
I think we should set a name in the example, also I think the `nil` is still from the options struct.